<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QMainWindow" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #ffff66;
    border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tinkoff API Trade Helper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pokupka">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #20AA4C;  border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501; }</string>
    </property>
    <property name="text">
     <string>Заявка на покупку</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>312</width>
      <height>183</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCalendarWidget { border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501;}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Колличество лотов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prodaja">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #AA2F20;  border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501; }</string>
    </property>
    <property name="text">
     <string>Заявка на продажу</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Название бумаги</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="vivod">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>291</y>
      <width>541</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser { border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501;}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'News701 BT'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="infa">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #f7943c;  border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501; }</string>
    </property>
    <property name="text">
     <string>Информация об аккаунте</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ema">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #f7943c;  border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501; }</string>
    </property>
    <property name="text">
     <string>Построение графика EMA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #D6D607;  border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501; }</string>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501;}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #D6D607;  border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501; }</string>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="paste">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501;}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Название компании, поиск FIGI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchb">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #20AA4C;  border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501; }</string>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Задать FIGI для графика</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="paste_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501;}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #D6D607;  border-style: outset;
    border-width: 2.3px;
    border-radius: 10px;
    border-color: #050501; }</string>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTinkoff_API">
    <property name="styleSheet">
     <string notr="true">QMenu { background-color: #ffb337; border: none; }</string>
    </property>
    <property name="title">
     <string>Tinkoff API</string>
    </property>
   </widget>
   <addaction name="menuTinkoff_API"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
